pkg/Cpanel/Security/Advisor/Assessors/Imunify360.pm

```diff
@@ -142,15 +128,41 @@ sub _process_template {
     die "Template processing failed: $output";
 }
 
+sub create_purchase_link {
+    my ($self) = @_;
+
+    my $imunify    = Whostmgr::Imunify360->new();
+    my $installed  = $imunify->is_product_installed();
+    my $custom_url = $imunify->get_custom_url();
+    my $price      = $imunify->get_product_price();
+    my $cp_url     = $self->base_path('scripts12/purchase_imunify360_init');
+
+    if ($custom_url) {
+        return locale()->maketext( '[output,url,_1,Get Imunify360,_2,_3].', $custom_url, 'target', '_blank', );
```

Comment:
I suspect these phrases are already in our queue but just so y’all know they can be simplified by passing in a hashref of attributes:

`return locale->maketext( '[output,url,_1,Get Imunify360,_2].', $custom_url, { target => '_blank' } );`

Reply:
It looks like roughly the same amount of code either way, but if you think the way you suggested is cleaner, I'll make the change.

Reply:
it just allows for more than one attribute (or none) so its a bit more reusable. no biggy.